src/brsTypes/components/RoSGNode.ts

```diff
@@ -1409,18 +1415,90 @@ export class RoSGNode extends BrsComponent implements BrsValue, BrsIterable {
         },
     });
 
+    /**
+     * Starting with a leaf node, traverses upward through the parents until it reaches
+     * a node without a parent (root node). Returns the parent chain starting with root-most parent.
```

Comment:
Consider adding your "returns the …" comment to an `@returns` attribute in JSDoc — it'll show up in fancy places thanks to `tsserver` 💰